data/ecosystem/integrations.yaml

```diff
@@ -217,3 +217,13 @@
   docsUrl: https://masstransit.io/documentation/configuration/observability
   components: [.NET]
   oss: true
+- name: Kuma
```

Comment:
to fix this add Kuma in line one (the cspell ignore section)

Reply:
updated